sdk/test-utils/recorder/src/utils/utils.ts

```diff
@@ -289,6 +289,10 @@ export interface RecorderStartOptions {
    * Generated recordings are updated by the "proxy-tool" based on the sanitizer options provided.
    */
   sanitizerOptions?: SanitizerOptions;
+  /**
+   * When a service uses a custom SSL certificate to communicate with the client.
+   */
+  tlsValidationCert?: string;
```

Comment:
Thanks for the change, @xirzec 🍏

Test proxy supports more than this. Adding here for future reference.
https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md#recording-options

Reply:
Do you mind adding a line in the changelog? 😎